spec/helpers/value_helper_spec.rb

```diff
@@ -31,4 +31,25 @@
       expect(helper.links_to_collections(args)).to have_css 'br'
     end
   end
+
+  describe 'originInfo_date_created_tesim' do
+    it 'does not modify a year string' do
+      field = 'originInfo_date_created_tesim'
+      value = ['1966', 'anything_after_first_element_ignored']
+      document = SolrDocument.new({ field => value })
+      args = { document: document, field: field }
+      expect(helper.value_for_originInfo_date_created_tesim(args))
+        .to eq(value.first)
+    end
+    it 'does normalize a time stamp string' do
+      now_utc = Time.now.utc.to_s
+      now_loc = Time.parse(now_utc).localtime.strftime '%Y.%m.%d %H:%M%p'
```

Comment:
How are you supposed to know the offset when this code is run and could be run on any machine anywhere?  The whole point of not specifying the offset is to use the local machine time.  Sorry, don't agree with houndci on this one.